### PR TITLE
‘value’ can be set to ‘null’

### PR DESCRIPTION
**Pull request by Baggz**

### CODE REVIEW

**Comment by XVincentX** [comment]
@Baggz Please attach the Trello card.

**Comment by Baggz** [comment]
@XVincentX 

https://trello.com/c/AkNxZ4jG/250-swagger-formdata-parameters-produce-errors-while-rendering-attributes-in-the-documentation

File: src/Components/PrimitiveValue/PrimitiveValue.js [hunk 1]
```diff
@@ -63,10 +63,16 @@ class PrimitiveValue extends React.Component {
   }
 
   render() {
+    if (this.props.value === null) {
```
Comment (XVincentX) [inline src/Components/PrimitiveValue/PrimitiveValue.js]: This check is almost the same you're performing few lines below (`isUndefined`). Could you please unify these?

Moreover, a `Primitive` value should always be not null so - these checks should probably happen in the container component, but that's another story

**Comment by Baggz** [comment]
We have discussed the issue with `this.props.value === null` offline. `typeof null === "object"` for that reason `isUndefined` won't catch `null` values. For now we'll keep it as it is, plan for the future is to make sure Refract structure coming from various adapters is consistent.

**Pull request merged**